content/Main_Notebooks/2 - Pandas Data Structures.ipynb

```diff
@@ -2345,12 +2345,29 @@
+    }
+   },
    "cell_type": "markdown",
    "id": "53d40244",
    "metadata": {},
    "source": [
     "### Combining Dataframes\n",
-    "One thing that you do often need to do when you make dataframes is merge them together. We will need some very small dataframes to see this behavior in."
+    "\n",
+    "When you load your data from the workbench, there is a high likelihood that you will have to combine two pre-existing dataframes together to make a new dataframe. For instance, there may be a dataframe representing people, and another dataframe representing medication prescriptions. Before combining these dataframes, each row in the person dataframe will represent a different person. The medication prescriptions dataframe will have one column with a person_id, and other information in other columns with information on medications that someone is prescribed, such as the medication name, and prescription date. To use information from both of these tables at once, you will need to join or merge by the person_id column that exists in both dataframes.![image.png](attachment:91ccf989-69cc-47c0-9922-ff457bb3c503.png)![image.png](attachment:7d9c480e-0666-4637-8fab-af3976cb8400.png)![image.png](attachment:79d3f474-dca4-498e-b295-8c96670415a0.png)![image.png](attachment:290762c9-2835-45fe-b5a8-46c13611dba3.png)\n",
```

Comment:
I'm not sure what happened, but it looks like image versions of the text was inserted here.

Reply:
Weird, I didn't notice this before. I'll remove the screenshots - it looks like they were intended to be temporary.